csharp/src/Ice/Communicator.cs

```diff
@@ -210,6 +215,8 @@ internal CancellationToken CancellationToken
 
         private static string[] _emptyArgs = Array.Empty<string>();
 
+        private static readonly Dictionary<string, IPluginFactory> _pluginFactories = new();
```

Comment:
This is no longer required, it was just to hold the factories added using the old API

Reply:
Fixed.